testutil/directory.go

```diff
@@ -143,3 +146,43 @@ func DirSize(path string) (int64, error) {
 	})
 	return size, err
 }
+
+// DirHash returns a hash of all files attribites and their content within a directory.
+func DirHash(path string) ([]byte, error) {
```

Comment:
As this is a `testutil`, it can take `testing.TB` object and call `Ok(...)` method for errors instead of `err != nil` checks and returning of error.

Reply:
updated, also updated the DirSize func to make it consistent.